### PR TITLE
add getSentenceVector to java api

### PR DESCRIPTION
**Pull request by benman1**
Update fb fastText cpp repo to latest. This includes the getSentenceVector.
Include the getSentenceVector method in java interface.

### CODE REVIEW

File: src/main/cpp/fasttext_wrapper.h [hunk 1]
```diff
@@ -27,6 +27,7 @@ namespace FastTextWrapper {
         std::vector<std::pair<fasttext::real,std::string>> predictProba(const std::string&, int32_t);
         std::vector<std::pair<fasttext::real,std::string>> predictProba(const std::string&, int32_t, fasttext::real);
         std::vector<fasttext::real> getWordVector(const std::string&);
+        std::vector<fasttext::real> getSentenceVector(const std::string& word);
```
Comment (lidalei) [inline src/main/cpp/fasttext_wrapper.h]: Could you remove `word`?

File: src/main/java/com/github/jfasttext/JFastText.java [hunk 2]
```diff
@@ -146,6 +146,15 @@ public ProbLabel predictProba(String text) {
     return wordVec;
   }
 
+  public List<Float> getSentenceVector(String word) {
```
Comment (lidalei) [inline src/main/java/com/github/jfasttext/JFastText.java]: `word` should be `text` or `sentence`?

File: src/test/java/com/github/jfasttext/JFastTextTest.java [hunk 3]
```diff
@@ -155,4 +155,14 @@ public void test10LoadDefaultLanguageDetectionModel() {
     System.out.println("Unloading model ...");
     jft.unloadModel();
   }
+
+  @Test
```
Comment (lidalei) [inline src/test/java/com/github/jfasttext/JFastTextTest.java:159]: Just out of curiosity, is there a way we can compare result in official Python API and JFastText?

**Comment by lidalei** [comment]
Sorry, I forgot to submit review.

File: src/test/java/com/github/jfasttext/JFastTextTest.java (hunk 3 above)
Comment (benman1) [reply src/test/java/com/github/jfasttext/JFastTextTest.java:159]: Just load a model and get the vector for a sentence/word. I've done it - it looks very similar between python and JFastText.

Comment (lidalei) [reply src/test/java/com/github/jfasttext/JFastTextTest.java:159]: 👍

**Comment by benman1** [comment]
I've changed the variable names as requested. Please accept my PR.

**Pull request merged**